### PR TITLE
[VRGTR] Van Richten's Guide To Ravenloft

### PR DESCRIPTION
**Pull request by kgirtxd**

### CODE REVIEW

File: supplements/van-richtens-guide-to-ravenloft/warlock-undead.xml [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+			<p class="indent">While projecting your spirit, you gain the following benefits:</p>
+			<ul>
+				<li>Your spirit and body gain resistance to bludgeoning, piercing, and slashing damage.</li>
+				<li>When you cast a spell of the conjuration or necromancy school, the spell doesn’t require verbal or somatic components or material components that lack a gold cost.</li>
+				<li>You have a flying speed equal to your walking speed and can hover. You can move through creatures and objects as if they were difficult terrain, but you take 1d10 force damage if you end your turn inside a creature or an object.</li>
+				<li>While you are using your Form of Dread, once during each of your turns when you deal necrotic damage to a creature, you regain hit points equal to half the amount of necrotic damage dealt.</li>
+			</ul>
+		</description>
+		<sheet action="Action" usage="1/Long Rest">
+			<description>As an action, you can project your spirit from your body. The body you leave behind is unconscious and in a state of suspended animation.&#13;
```
Comment (NickVendel) [inline supplements/van-richtens-guide-to-ravenloft/warlock-undead.xml:126]:
```suggestion
			<description>You can project your spirit from your body. The body you leave behind is unconscious and in a state of suspended animation.&#13; 
```

**Review by NickVendel — CHANGES_REQUESTED** [review]

File: supplements/van-richtens-guide-to-ravenloft/backgrounds.xml [hunk 2]
```diff
@@ -0,0 +1,184 @@
...
+				<tr><td>6</td><td>You were cursed with lycanthropy and later cured. You are now haunted by the innocents you slaughtered.</td></tr>
+				<tr><td>7</td><td>A hag kidnapped and raised you. You escaped, but the hag still has a magical hold over you and fills your mind with evil thoughts.</td></tr>
+				<tr><td>8</td><td>You opened an eldritch tome and saw things unfit for a sane mind. You burned the book, but its words and images are burned into your psyche.</td></tr>
+				<tr><td>9</td><td>A fiend possessed you as a child. You were locked away but escaped. The fiend is still inside you, but now you try to keep it bottled up.</td></tr>
+				<tr><td>10</td><td>You did terrible things to avenge the murder of someone you loved. You became a monster, and it haunts your waking dreams.</td></tr>
+			</table>
+			<div element="ID_WOTC_VRGTR_BACKGROUND_FEATURE_HEART_OF_DARKNESS" />
+		</description>
+		<setters>
+			<set name="short">2 Arcana / Investigation / Religion / Survival, Language, Exotic Language </set>
```
[the remark this answers is not in the document]
Comment (kgirtxd) [reply supplements/van-richtens-guide-to-ravenloft/backgrounds.xml:95]: We can update those others later

**Review by NickVendel — CHANGES_REQUESTED** [review]

**Comment by FireWallach** [comment]
Looking forward to this, thanks for your hard work!

File: supplements/van-richtens-guide-to-ravenloft/items.xml [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+		<update version="0.0.1">
+			<file name="items.xml" url="https://raw.githubusercontent.com/AuroraLegacy/elements/master/supplements/van-richtens-guide-to-ravenloft/items.xml" />
+		</update>
+	</info>
+
+	<element name="Harkon’s Bite" type="Magic Item" source="Van Richten's Guide To Ravenloft" id="ID_WOTC_VRGTR_MAGIC_ITEM_HARKONS_BITE">
+		<description>
+			<p>A dire wolf tooth dangles from this simple cord necklace. While you wear it, the necklace grants you a +1 bonus to ability checks and saving throws.</p>
+			<p class="indent"><strong><em>Curse. </em></strong>Attuning to Harkon’s Bite curses you until either Harkon Lukas removes the necklace from you or you are targeted by a <em>remove curse</em> spell or similar magic. As long as you remain cursed, you cannot remove the necklace.</p>
+			<p>Upon donning or removing the necklace, whether you are attuned to it or not, you are afflicted with <em>werewolf lycanthropy</em> as detailed in the <em>Monster Manual</em>. The curse lasts until the dawn after the next full moon. If you are still wearing the necklace at this time, you are afflicted with the lycanthropy again.</p>
```
Comment (NickVendel) [inline supplements/van-richtens-guide-to-ravenloft/items.xml:14]: book doesn't seem to have words "werewolf lycanthropy" in italics.
```suggestion
			<p>Upon donning or removing the necklace, whether you are attuned to it or not, you are afflicted with werewolf lycanthropy as detailed in the <em>Monster Manual</em>. The curse lasts until the dawn after the next full moon. If you are still wearing the necklace at this time, you are afflicted with the lycanthropy again.</p>
```

File: supplements/van-richtens-guide-to-ravenloft/items.xml [hunk 4]
```diff
@@ -0,0 +1,99 @@
...
+		<name>Items and Weapons</name>
+		<update version="0.0.1">
+			<file name="items.xml" url="https://raw.githubusercontent.com/AuroraLegacy/elements/master/supplements/van-richtens-guide-to-ravenloft/items.xml" />
+		</update>
+	</info>
+
+	<element name="Harkon’s Bite" type="Magic Item" source="Van Richten's Guide To Ravenloft" id="ID_WOTC_VRGTR_MAGIC_ITEM_HARKONS_BITE">
+		<description>
+			<p>A dire wolf tooth dangles from this simple cord necklace. While you wear it, the necklace grants you a +1 bonus to ability checks and saving throws.</p>
+			<p class="indent"><strong><em>Curse. </em></strong>Attuning to Harkon’s Bite curses you until either Harkon Lukas removes the necklace from you or you are targeted by a <em>remove curse</em> spell or similar magic. As long as you remain cursed, you cannot remove the necklace.</p>
```
Comment (NickVendel) [inline supplements/van-richtens-guide-to-ravenloft/items.xml:13]:
```suggestion
			<p class="indent"><strong><em>Curse. </em></strong>Attuning to <em>Harkon’s Bite</em> curses you until either Harkon Lukas removes the necklace from you or you are targeted by a <em>remove curse</em> spell or similar magic. As long as you remain cursed, you cannot remove the necklace.</p>
```

**Review by NickVendel — CHANGES_REQUESTED** [review]

**Review by NickVendel — APPROVED** [review]

**Review by AngryGuineaPig — APPROVED** [review]

**Pull request merged**